### PR TITLE
Financial Assistance Page under Events

### PR DESCRIPTION
**Pull request by railsstudent**
#120

### CODE REVIEW

**Comment by railsstudent** [comment]
Add marketing team to provide content. I am totally BSing here

**Comment by railsstudent** [comment]
selection_025
selection_026

File: assets/pages/events/financial-assistance.jinja [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        </div>
+    </div>
+</div>
+  <div class="container">
+      <div class="content">
+        <p>Hong Kong OpenSource Conference provides financial assistance to selected individuals to attend Hong Kong OpenSource Conference. 
+          The purpose of financial assistance is to give opportunities to local open source developers and supporters access to our region's 
+          premier Open Source event. 
+        </p>
+        <a class="waves-effect waves-light z-depth-2 btn" href="https://docs.google.com/forms/d/e/1FAIpQLSce3Oq5hy8KNYN_RakbJWlbzzp_GpIh_pCXtyvp7SLovc-5Yg/viewform?c=0&w=1">Apply</a>
```
Comment (tonyhhyip) [inline assets/pages/events/financial-assistance.jinja]: Use link: https://goo.gl/forms/rpNLFNeHQ2jPv7Uj2

File: assets/pages/events/financial-assistance.jinja [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<div class="theme-area">
+    <div class="content">
+        <div class="container">
+            <h1>Financial Assistance Application</h1>
+        </div>
+    </div>
+</div>
+  <div class="container">
+      <div class="content">
+        <p>Hong Kong OpenSource Conference provides financial assistance to selected individuals to attend Hong Kong OpenSource Conference.
```
Comment (tonyhhyip) [inline assets/pages/events/financial-assistance.jinja]: Please change the name to "Hong Kong Open Source Conference"

**Review by tonyhhyip — CHANGES_REQUESTED** [review]

**Comment by railsstudent** [comment]
selection_013

**Comment by railsstudent** [comment]
selection_016

**Review by tonyhhyip — APPROVED** [review]

**Comment by tonyhhyip** [comment]
As it is based on #116, need to wait for review in #116

**Pull request merged**